src/formatDate.js

```diff
@@ -43,14 +43,53 @@
  * ) // '18.02.1997'
  *
  * @param {string} date
- * @param {string[]} fromFormat
+ * @param {string[]} string
  * @param {string[]} toFormat
  *
  * @returns {string}
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const string = fromFormat.join('.');
```

Comment:
Why do you need to join the string? You can check by indexOf inside the original array.